src/translations/index.ts

```diff
@@ -0,0 +1,19 @@
+import { defaultLocale } from "translations/config";
+import { Locale } from "translations/types";
+import en from "./locales/en.json";
+import pl from "./locales/pl.json";
+import common from "./locales/common.json";
+
+const translations = {
```

Comment:
logic for managing translations is simplified a little bit, no need to some custom handling for keys - library can do that for us